service/haproxy/run

```diff
@@ -38,7 +38,11 @@ reload() {
   socat /var/run/haproxy/socket - <<< "show servers state" > /var/state/haproxy/global
 
   # Trigger reload
-  haproxy -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $(pidof haproxy)
+  HAPROXY_PIDS=$(pidof haproxy)
+  haproxy -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $HAPROXY_PIDS
+  if [ -n "${HAPROXY_RELOAD_SIGTERM_DELAY-}" ]; then
+    sleep $HAPROXY_RELOAD_SIGTERM_DELAY && kill $HAPROXY_PIDS 2> /dev/null &
```

Comment:
- Is this PR applicable only when we explicitly specify the reload command? Because the reload function [here](https://github.com/mesosphere/marathon-lb/blob/f7ca0e03e6396b96fcd3f6b3464fb06ee3e33dc6/marathon_lb.py#L631) still uses default commands to reload if we don't specify anything explicitly.

- Also we need to explicitely specify the haproxy config file path as /marathon-lb/haproxy.cfg as per [this](https://github.com/mesosphere/marathon-lb/blob/f7ca0e03e6396b96fcd3f6b3464fb06ee3e33dc6/service/haproxy/run#L22) ?